tests/test_soak.py

```diff
@@ -10,7 +10,7 @@
 
 
 LOGGER = logging.getLogger(__name__)
-SOAK_SPARK_APP_NAME='/spark'
+SOAK_SPARK_SERVICE_NAME = os.getenv('SOAK_SPARK_SERVICE_NAME', '/spark')
```

Comment:
this is un utils.py already isn't it?

Reply:
@ArtRand No, only the package name is in utils.py. The service name varies by test, so it's not normally read from the environment. I think only in the soak case do we want to read the service name from the environment (because in the soak cluster we're not constantly installing and uninstalling Spark).